apps/app/src/views/routes/Order.tsx

```diff
@@ -339,354 +373,446 @@ export const Order = () => {
+            <Card
+              p={4}
+              bg={theme.colors.slate['50']}
+              borderWidth="1px"
+              borderColor={theme.colors.slate['200']}
+              borderRadius="lg"
+              variant="outline"
+            >
+              <VStack spacing={4}>
+                <Box w="100%">
+                  <Text fontSize="xs" color={theme.colors.slate['700']}>
+                    PATIENT
+                  </Text>
+                  <Text fontSize="sm" fontWeight="medium">
+                    {order?.patient && order.patient.name.full}
+                  </Text>
+                </Box>
+                <VStack w="100%" align="left">
+                  <Text fontSize="xs" color={theme.colors.slate['700']}>
+                    PRESCRIPTION
+                  </Text>
+                  {fills?.length > 0 &&
+                    fills.map((fill) => (
+                      <Box w="100%" key={fill.treatment.name}>
+                        <Text fontSize="sm" fontWeight="medium">
+                          {fill?.treatment?.name}
+                        </Text>
+                        <Text fontSize="sm" color={theme.colors.slate['500']}>
+                          {formatRxString({
+                            dispenseQuantity: fill?.prescription?.dispenseQuantity,
+                            dispenseUnit: fill?.prescription?.dispenseUnit,
+                            fillsAllowed: fill?.prescription?.fillsAllowed,
+                            instructions: fill?.prescription?.instructions
+                          })}
+                        </Text>
+                      </Box>
+                    ))}
+                </VStack>
+              </VStack>
+            </Card>
+          </>
+        }
+        prependContext={
+          !order
+            ? ''
+            : `
+Order:
+  ID: ${order.id}
+
+----
+Patient:
+  ID: ${order?.patient?.id} 
+  Name: ${order?.patient?.name?.full}
+
+----
+Prescriptions:
+${fills.map(
+  (fill) => `
+  Name: ${fill.treatment.name}
+  Info: ${formatRxString({
+    dispenseQuantity: fill?.prescription?.dispenseQuantity,
+    dispenseUnit: fill?.prescription?.dispenseUnit,
+    fillsAllowed: fill?.prescription?.fillsAllowed,
+    instructions: fill?.prescription?.instructions
+  })}
+`
+)}
+
+---- 
+Description: 
+        `
+        }
+      />
+      <Page
+        header="Order"
+        buttons={
+          <HStack>
+            <Button
+              aria-label="Cancel Order"
+              variant="outline"
+              borderColor="red.500"
+              textColor="red.500"
+              colorScheme="red"
+              isLoading={updating}
+              loadingText="Canceling..."
+              isDisabled={
+                loading ||
+                order.state === types.OrderState.Canceled ||
+                order.state === types.OrderState.Completed ||
+                order?.fulfillment?.state === 'SHIPPED'
```

Comment:
Shouldnt it be hidden if these are the case?

Reply:
This is so that in some states the button is mysteriously gone, leaving the user confused. a better version of this also explains why it's disabled.